src/Common.am

```diff
@@ -73,3 +74,21 @@ COMPAT_LIB = $(top_builddir)/compat/libcompatsquid.la $(LIBPROFILER)
 
 ## Some helpers are written in Perl and need the local shell defined properly
 subst_perlshell = sed -e 's,[@]PERL[@],$(PERL),g' <$(srcdir)/$@.pl.in >$@ || ($(RM) -f $@ ; exit 1)
+
+EXTENSIONS=.hdrtest
+.h.hdrtest:
+	@srcfilepath=`echo $< | $(SED) 's%^$(top_srcdir)/%%'`; \
```

Comment:
Here and elsewhere, please use camelCase for new shell variables. IIRC, we have discussed why already, but I can repeat those arguments if needed.

Reply:
ok